frontend/src/components/grantaccess/GrantDelegatedAdmin.vue

```diff
@@ -111,6 +112,7 @@ const confirmSubmit = async () => {
     const username = formData.value.userId.toUpperCase();
     const role = getSelectedRole()?.name;
     const successList: string[] = [];
+    const newlyAddedList: number[] = [];
```

Comment:
Same here, maybe `newDelegatedAdminAccessIds` or something similar or at least add a comment for variable to explain what it holds.